src/index.js

```diff
@@ -1,60 +1,103 @@
+import {loadPeople, loadPlanets, loadFilms, loadSpecies, loadVehicles} from './loadData.js';
+
+let category;
+let categorySelected = false;
+let currentPageInfo;
+
 const cardsField = document.querySelector('.card-body');
 const categories = document.querySelectorAll(`.nav-link`);
 const paginator = document.createElement('ul');
-const loader = document.querySelector('.starwars-demo')
-let category;
+const introImage = document.querySelector('.starwars-demo');
+const loader = document.querySelector('.lds-facebook');
+const search = document.getElementById('search');
+const searchField = document.querySelector('.search-field');
+const save = document.getElementById('save');
+const load = document.getElementById('load');
+const themeSwitcher = document.getElementById('customSwitch1');
```

Comment:
Avoid to use digits in names

Reply:
I don't see digits in the names here. Can you write more specifically?

Reply:
`customSwitch1` - you have 1 at the end  ^_^

Reply:
Now i see it! Its custom id for this bootstrap switcher, changed.